### ISSUE TITLE
Please add a “time” format

### ISSUE THREAD

**Issue by stapelberg** [prometheus/promdash#298]
Currently, we can chose between “base 1024 KMGTP”, “KMBT” and “numeric”. Aside from the fact that the first two are not clear to me (add a hover text with an explanation?), I’d like to be able to specify that my data is in a unit of time, e.g. milliseconds, and have the graph automatically label values in the appropriate unit, i.e. milliseconds,  seconds, minutes, …, depending on the values.

The use case is to display latency values.

Thanks for considering this :).

**Comment by brian-brazil** [comment]
Over in console templates we assume everything is seconds/bytes, and have humanizing functions http://prometheus.io/docs/visualization/template_reference/#numbers (the graphs do similar). Maybe similar can be done here?

**Comment by stapelberg** [comment]
Definitely, but note that seconds really is not fine-grained enough for latency metrics: knowing whether round-trip times are below one second is a nice start, but I’d really want to know if it’s 10ms or 900ms :).

**Comment by brian-brazil** [comment]
Everything is a float in prometheus, I've got things rendering as `us` in places via this method.

**Comment by juliusv** [comment]
Assuming that everything is in seconds is (for better or worse) at odds with the community decision we had a while ago (weird, I can't find the thread in Google Groups anymore, although it's in my inbox) about units, and we have a lot of existing metrics which are milliseconds, microseconds, etc. So we'll need to be able to configure the scale of the unit explicitly.
